src/models/stackedAreaChart.js

```diff
@@ -375,11 +375,15 @@ nv.models.stackedAreaChart = function() {
                 //If we are in 'expand' mode, force the format to be a percentage.
                 var valueFormatter = (stacked.style() == 'expand') ?
                     function(d,i) {return d3.format(".1%")(d);} :
-                    function(d,i) {return yAxis.tickFormat()(d); };
+                    tooltip.valueFormatter(); 
```

Comment:
Okay. I was under the impression that interactiveLayer was something we wanted to abstract away for the user. Why does stackedAreaChart have tooltip by itself at all then? Is it deprecated?